### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by reazulhoque**
This should potentially update the Documentation with all the NumPy function we support. 

@PokhodenkoSA is there a way to preview the changes before we merge it?

### CODE REVIEW

**Comment by diptorupd** [comment]
>  is there a way to preview the changes before we merge it?

I suggest hosting the generated pages under your personal github.io page. It will be really useful to review by looking at the resulting documentation.

**Comment by reazulhoque** [comment]
Preview: https://reazulhoque.github.io/numba-dppy/latest/index.html

**Comment by diptorupd** [comment]
> 
> 
> Preview: https://reazulhoque.github.io/numba-dppy/latest/index.html

@reazulhoque following up from out conversation from yesterday, it would be great to have a table for the functions that Numba supports on CPU and what we support for XPU.

File: dev/_static/documentation_options.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var DOCUMENTATION_OPTIONS = {
     URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
-    VERSION: '0+untagged.653.g5ea019b',
+    VERSION: '0+untagged.655.g2343fe5',
```
Comment (PokhodenkoSA) [inline dev/_static/documentation_options.js:3]: Why this files and others from `dev` folder are committed?

**Review by PokhodenkoSA — CHANGES_REQUESTED** [review]
Please remove temporary files.

**Comment by PokhodenkoSA** [comment]
@reazulhoque as I know we have automatic documentation generation - #547. So it is better to make PR for changing documentation sources rather than generated documentation.

**Pull request closed without merging**